### ISSUE TITLE
Azure Traffic Collector CLI

### ISSUE THREAD

**Issue by jaganpani** [Azure/azure-cli#22270]
**Resource Provider**
Microsoft.NetworkFunction

**Description of Feature or Work Requested**
Generate CLI for Azure Traffic Collector

**Minimum API Version Required**
2022-05-01

**Swagger Link**
https://github.com/Azure/azure-rest-api-specs-pr/pull/4405 (In Dev branch.)
Working on merging swagger to public repo.

**Target Date**
May 2022 (Public Preview)

**Comment by yonzhan** [comment]
PLR

**Comment by jsntcy** [comment]
@jaganpani, once the swagger is merged into public repository, please let us know, thanks.

**Comment by jaganpani** [comment]
@jsntcy any updates on this request.

**Comment by jsntcy** [comment]
@jaganpani, recently we developed a new CLI code generator and we'd like to invite you (or someone from your team) as our first customer to try our new code generator to generate CLI code automatically.
We can set up a quick meeting with you to guide you to generate code step by step using the tool.
Really appreciated if you can try our new codegen tool.

**Comment by jaganpani** [comment]
@jsntcy , lets setup a meeting to go over the tool.

**Comment by jsntcy** [comment]
> @jsntcy Yu Chen FTE , lets setup a meeting to go over the tool.

My teammate Kai Ru will setup a meeting with you to go over the tool.

**Comment by jsntcy** [comment]
@jaganpani, what's release stage for traffic collector? Public Preview or GA?

**Comment by yonzhan** [comment]
@jaganpani, any update?

**Comment by chasewilson** [comment]
After talking with the Service team, would like to GA along with the **Microsoft Ignite release**.  This might be a good candidate for CodeGen V2 and the team has expressed interest in using CodeGen V2 as long as it is stable and won't interfere with their timeline. 

The team is also requesting information about timelines for release. Branch cutoff etc. @yonzhan, let me know if there are any questions or concerns about any of these points.

**Comment by yonzhan** [comment]
@chasewilson good to know. For new RP like this, we encourage them using CodeGen V2 to implement.
